### ISSUE TITLE
Please save tooltip sorting method and panel open/closed layout in a cookie so that I don't have to reset it every time I reload TensorBoard due to a UI glitch or memory leak.

### ISSUE THREAD

**Issue by iansf** [tensorflow/tensorboard#1321]
When running TB with a lot of different runs, I always use Nearest for the tooltip sorting -- it makes it much easier to locate the particular run I'm interested in on a busy graph. However, every time I have to refresh the page, the sorting method reverts back to Default. I understand that the sorting method isn't in the url, since that tends to be a personal preference, but it does seem like quite a good idea to save the personal preference like most of the other things are saved.

In the same vein, if I have to reload TB (which is often the case because of memory use and UI glitches), it loses all of the open/closed states of the panels in each tab, and reverts back to having the top three panels open and the rest closed.

Thus, my user experience upon reloading is: close all of the panels that are open, hopefully before the graph loading pegs my cpu; set the tooltip sorting to Nearest, reopen the panels I had open before I reloaded. None of those steps would be necessary if these two pieces of state were saved in the cookie.

**Comment by vnicolici** [comment]
I agree, this is extremely annoying, especially on mobile browsers that automatically refresh the page when you switch to another app. The entire layout and all the settings should be saved.
